zetaclient/evm/evm_signer.go

```diff
@@ -32,6 +37,26 @@ import (
 	zbridge "github.com/zeta-chain/zetacore/zetaclient/zetabridge"
 )
 
+type TransactionData struct {
+	srcChainID *big.Int
+	toChainID  *big.Int
+	sender     ethcommon.Address
+	to         ethcommon.Address
+	asset      ethcommon.Address
+	amount     *big.Int
+	gasPrice   *big.Int
+	gasLimit   uint64
+	message    []byte
+	sendHash   [32]byte
+	nonce      uint64
+	height     uint64
+
+	cmd            string
+	params         string
+	outboundParams *types.OutboundTxParams
+	flags          observertypes.CrosschainFlags
```

Comment:
Yes, I noticed this as well as I was reviewing again. It doesn't need to be passed to any of the signers. I removed it and queried it from zetabridge directly. We can create another issue for this because it might be out of the scope of this PR.